source/utilities.tex

```diff
@@ -14819,39 +14846,60 @@
 string s4 = format("{:*^6}", 'x');      // value of \tcode{s4} is \tcode{"**x***"}
 string s5 = format("{:6d}", c);         // value of \tcode{s5} is \tcode{"\ \ \ 120"}
 string s6 = format("{:6}", true);       // value of \tcode{s6} is \tcode{"true\ \ "}
+string s7 = format("{:*<6.3}", "123456");   // value of s7 is "123***"
```

Comment:
I think the comments should be aligned. Those from the added strings and the preexisting ones. Just like the paper.
![1677093953](https://user-images.githubusercontent.com/21071787/220737457-e16693dd-ec95-4de5-bf18-29b87046da97.png)

Reply:
ok, done